Mathlib/Probability/IdentDistrib.lean

```diff
@@ -366,4 +366,19 @@ theorem Memℒp.uniformIntegrable_of_identDistrib {ι : Type*} {f : ι → α 
 
 end UniformIntegrable
 
+/-- If `X` and `Y` are independent and `(X, Y)` and `(X', Y')` are identically distributed,
+then `X'` and `Y'` are independent. -/
+lemma indepFun_of_identDistrib_pair
+    {μ : Measure γ} {μ' : Measure δ} [IsFiniteMeasure μ] [IsFiniteMeasure μ']
+    {X : γ → α} {X' : δ → α} {Y : γ → β} {Y' : δ → β} (hX : AEMeasurable X μ)
+    (hX' : AEMeasurable X' μ') (hY : AEMeasurable Y μ) (hY' : AEMeasurable Y' μ')
```

Comment:
The AEMeasurable assumptions are unnecessary as you can get them from `IdentDistrib.aemeasurable_fst`, e.g.
```lean
example {μ : Measure γ} {μ' : Measure δ} [IsFiniteMeasure μ] [IsFiniteMeasure μ']
    {X : γ → α} {X' : δ → α} {Y : γ → β} {Y' : δ → β}
    (h_ident : IdentDistrib (fun ω ↦ (X ω, Y ω)) (fun ω ↦ (X' ω, Y' ω)) μ μ') :
    AEMeasurable X μ := by
    rw [(by ext; simp : X = Prod.fst ∘ (fun ω ↦ (X ω, Y ω)))]
    refine AEMeasurable.comp_aemeasurable measurable_fst.aemeasurable h_ident.aemeasurable_fst
```

Reply:
Done. Thank you very much for your review.